rosdep/python.yaml

```diff
@@ -4047,6 +4047,15 @@ python3-pep8:
 python3-pkg-resources:
   debian: [python3-pkg-resources]
   ubuntu: [python3-pkg-resources]
+python3-pyparsing:
+  arch: [python-pyparsing]
+  debian: [python3-pyparsing]
+  fedora: [python3-pyparsing]
+  gentoo: [dev-python/pyparsing]
+  ubuntu: [python3-pyparsing]
+python3-pytest:
+  arch: [python-pytest]
+  ubuntu: [python3-pytest]
```

Comment:
We encourage to add rules for debian and not only ubuntu.
arch fedora and gentoo are optional but apreciated